src/lib/annotations/Annotator.scss

```diff
@@ -15,6 +15,72 @@ $avatar-color-7: #159f45;
 $avatar-color-8: #b800b2;
 $avatar-color-9: #f22c44;
 
+$tablet: 'max-width: 768px';
+
+.bp-annotate-draw-header {
+    background-color: $twos;
+}
+
+.bp-annotate-draw-undo-redo-container {
+    margin: 0 auto;
+}
+
+@media ($tablet) {
+    .bp-annotate-draw-undo-redo-container {
+        margin: 0;
+    }
+}
+
+.bp-btn-annotate-draw-cancel {
+    background-color: $twos;
+    border: solid $off-white 1px;
+    color: $off-white;
+
+    &:hover {
+        background-color: $black;
+    }
+}
+
+.bp-btn-annotate-draw-post {
+    background-color: $off-white;
+    color: $twos;
+
+    &:hover {
+        background-color: $white;
+    }
+}
+
+.bp-annotate-draw-post-cancel-container {
+    position: absolute;
+    right: 10px;
+}
+
+.bp-btn-annotate-draw-undo,
```

Comment:
What happens if you don't have :hover, :active, and :focus?
I'm just curious as to why those are included

Reply:
We need to override the normal button styles since they assume things like background/border for all of these states. The undo redo buttons are an svg in a clear button. I considered not using `'bp-btn' but we gain convenience with the disabled state, cursor, and using the correct hit box.

Reply:
@tonyjin have you dealt with anything like this before?

Reply:
Have you tried using 'bp-btn-plain`?

I'd do something like `class="bp-btn bp-btn-plain bp-btn-annotate-draw-undo"` - not sure if you need both `bp-btn` and `bp-btn-plain`